pkg/comp-functions/functions/common/instance_namespace.go

```diff
@@ -139,3 +153,70 @@ func createNamespacePermissions(ctx context.Context, instance string, instanceNs
 	}
 	return svc.SetDesiredKubeObjectWithName(roleBinding, instance+"-service-rolebinding", "namespace-permissions")
 }
+
+// DisableBilling deploys a special config map to the appcat control namespace.
+// This configMap contains a key that specifies if a given namespace should be billed or not.
+// The configMap can also be used for other configurations in the future.
+func DisableBilling(instanceNamespace string, svc *runtime.ServiceRuntime) error {
+	controlNS, ok := svc.Config.Data["controlNamespace"]
+	if !ok {
+		return fmt.Errorf("controlNamespace not specified in composition input")
+	}
+
+	objName := instanceNamespace + "-config"
+
+	checkCM := &corev1.ConfigMap{}
+	err := svc.GetDesiredKubeObject(checkCM, objName)
+	if err != nil && err != runtime.ErrNotFound {
+		return err
+	}
+
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      instanceNamespace,
+			Namespace: controlNS,
+		},
+		Data: map[string]string{
+			billingCMKey: "true",
```

Comment:
This sets as disabled? If yes name of the key is confusing.

Reply:
Do you mean the name of the constant `billingCMKey`? Or the value of the constant `billingDisabled `?

Reply:
The constant, you might consider changing the name.